assets/js/timetree.js

```diff
@@ -8,119 +8,136 @@ import {schemeGreens} from "d3-scale-chromatic"
 const d3 = {select, forceSimulation, forceManyBody, forceCenter, forceCollide,
     forceLink, forceY, line, curveNatural, scaleSequential, schemeGreens};
 
+// load & parse leaf data from json embedded in the document
+const leafData = document.querySelector('.leaf-data');
+const data = JSON.parse(leafData.value);
+
```

Comment:
So we're not using jquery in general?

Reply:
right — built in js functionality is sufficiently advanced we can use it directly, we don't need the bloat/complication of jquery